src/routing/oldContentRouteGuard.ts

```diff
@@ -0,0 +1,31 @@
+
+import { RouteGuard, Route } from "@paperbits/common/routing";
+import { ViewManager } from "@paperbits/common/ui/viewManager";
+import { Toast } from "@paperbits/common/ui";
+
+export class OldContentRouteGuard implements RouteGuard {
+    private toast: Toast;
+
+    constructor(private readonly viewManager: ViewManager) { }
+
+    public async canActivate(route: Route): Promise<boolean> {
+        if (sessionStorage.getItem("OldContent") || this.toast) {
+            return true;
+        }
+
+        const main = window.document.getElementsByTagName("iframe");
+        if (main.length > 0) {
+            const iframe = main[0];
+            const deprecatedNavItems = iframe.contentDocument.body.getElementsByTagName("navbar");

```

Comment:
I would switch
`iframe.contentDocument.body.getElementsByTagName("navbar")`
to
`iframe.contentDocument.body.querySelector("nav.navbar-brand")`